.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: java
 jdk:
- - oraclejdk8
+ - oraclejdk9
```

Comment:
Does this mean Travis will run the tests using java 9?

Reply:
Yes, that's right. Everything would be compiled at java 8 compliance but the run time in travis would be 9.
Running with 9 JRE is more realistic, since it would be what you get by default when you install a jre in your system these days (or perhaps it's already 10).